src/Microsoft.Health.Fhir.SqlServer/Registration/FhirServerBuilderSqlServerRegistrationExtensions.cs

```diff
@@ -30,7 +33,22 @@ public static IFhirServerBuilder AddSqlServer(this IFhirServerBuilder fhirServer
             services.AddSqlServerBase<SchemaVersion>(configureAction);
             services.AddSqlServerApi();
 
-            services.Add(provider => new SchemaInformation((int)SchemaVersion.V1, (int)SchemaVersion.V2))
+            services.Add(provider => new SchemaInformation((int)SchemaVersion.V1, (int)SchemaVersion.V3))
+                .Singleton()
+                .AsSelf()
+                .AsImplementedInterfaces();
+
+            services.Add<SqlServerStatusRegistry>()
```

Comment:
Let's change this to scoped.
Inside, then we'll need to change the constructor here:
```
public SearchParameterStatusManager(
            Func<IScoped<ISearchParameterRegistry>> searchParameterRegistryFactory,
```
Invoked with:
```
using(var registry = searchParameterRegistryFactory.Invoke()) { ...registry.Value... }
```

Reply:
Thanks, fixed in 725eefb.